assets/js/base/context/test/utils.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Internal dependencies
+ */
+import { assertValidContextValue } from '../utils';
+
+describe( 'assertValidContextValue', () => {
+	const contextName = 'testContext';
+	const validationMap = {
+		cheesburger: {
```

Comment:
Missing `e`? :hamburger: 
```suggestion
		cheeseburger: {
```